### PR TITLE
Added support for CLI usage

### PR DESCRIPTION
**Pull request by skovhus**
I've added support for CLI usage, including dependencies and scoping.

Besides to follow the existing styleguide of the project, I added eslint. 

Besides I corrected two spelling mistakes in the README.

Hope you like it.

### CODE REVIEW

File: bin/scss-to-json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var minimist = require('minimist');
+
+var scssToJson = require('./../main.js');
```
Comment (TheSavior) [inline bin/scss-to-json]: Since the package.json specifies the main file, this should probably just be `require('../')`

File: bin/usage.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Usage: node ./bin/scss-to-json [options] [<file>]
```
Comment (TheSavior) [inline bin/usage.txt]: This should probably just be `Usage: node scss-to-json [options] [<file>]` and assume that people have it installed globally or know where the file is. Nobody will be running this command with `./bin/scss-to-json` though, which is why I don't think this makes sense.

**Comment by TheSavior** [comment]
Nice work and thanks for contributing!

We will be adding eslint support separately from this PR. All of the eslint changes makes this PR really large. Can you limit the scope of this PR to simply be adding the CLI? It is more likely to be merged that way.

**Comment by skovhus** [comment]
I split it up into:
https://github.com/ryanbahniuk/scss-to-json/pull/5 (please merge this first)
https://github.com/ryanbahniuk/scss-to-json/pull/6
https://github.com/ryanbahniuk/scss-to-json/pull/7

**Comment by ryanbahniuk** [comment]
Thanks @skovhus. I'll review each of them soon.

**Pull request closed without merging**